packages/elements/src/components/API/utils.ts

```diff
@@ -51,7 +51,19 @@ export const computeTagGroups = (serviceNode: ServiceNode) => {
   return { groups: orderedTagGroups, ungrouped };
 };
 
-export const computeAPITree = (serviceNode: ServiceNode) => {
+interface ComputeAPITreeConfig {
+  hideSchemas?: boolean;
+}
+
+const defaultComputerAPITreeConfig = {
+  hideSchemas: false,
+};
+
+export const computeAPITree = (serviceNode: ServiceNode, config: ComputeAPITreeConfig = {}) => {
+  const mergedConfig = {
+    ...defaultComputerAPITreeConfig,
+    ...config,
+  };
```

Comment:
This is a bit dangerous as a config like `{hideSchemas: undefined}` can override the default. Not a huge deal here as the default is `false`, but as a pattern in general, be aware of the problems. A safer tool is something like [_.defaults](https://lodash.com/docs/4.17.15#defaults) or similar property-by-property logic e.g. with a `??`.

Reply:
I used defaults.